templates/bundles/PrestaShopBundle/Admin/Sell/Catalog/Product/CategorySearch/category_search.html.twig

```diff
@@ -0,0 +1,40 @@
+<div id="product_catalog_category_tree_filter" class="d-inline-block dropdown dropdown-clickable mr-2 mb-2">
```

Comment:
As I suggested earlier I wonder if we could use the tree form the product form page, for now it can select multiple categories but we could add a limit in it (since I believe this filter should only select one category)